hardware/nic/bnx.pan

```diff
@@ -0,0 +1,7 @@
+structure template hardware/nic/bnx;
```

Comment:
You are probably better of using one of the templates under `hardware/nic/broadcom/`